modules/alloy/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added a new `preserve` option to `LayoutInside` which will preserve the previous placement inside the component.
 - Added the `alwaysFit` layout property which allows for the layout to specify if it should always claim to fit, no matter what.
 
+### Improved
+- Improved the performance of constructing event handlers and components.
+- Improved the performance of split toolbars when there were no groups to render.
```

Comment:
Changelogs should always be past tense.

Reply:
The sentence / changelog entry is past tense, it uses "Improved" instead of "Improves". The section after `when` should be present tense (or infinite / indefinite something), because it's describing a condition that can still happen.